services/client/package.json

```diff
@@ -13,7 +13,7 @@
     "build": "react-app-rewired build",
     "test": "react-app-rewired test --env=jsdom",
     "eject": "react-app-rewired eject",
-    "lint": "eslint ."
+    "lint": "eslint src"
```

Comment:
👍

Reply:
No, but I'll update it as originally I had assumed `.` was `src`.  Better to be specific here; given that `./src` works too